docs/index.html

```diff
@@ -371,340 +337,685 @@ <h3>RP Meta-data</h3>
+                The <a href="#rp">RP</a> MUST ignore unrecognized request parameters. Request and response parameters
+                MUST NOT be included more than once.
+            </p>
+
+            <p>
+                For example, the <a href="#siop">SIOP</a> makes the following HTTPS request:
+            </p>
+            <pre class="example nohighlight">
+                POST /device_verification HTTP/1.1
+                Host: server.example.com
+                Content-Type: application/json
+
+                {
+                  "device_code" : "GmRhmhcxhwAzkoEqiMEg_DnyEysNkuNhszIySk9eS",
+                  "user_code" : "WDJB-MJHT",
+                  "expires_in" : 1800
+                }
+            </pre>
+
+
+            <h4 id="md-device-binding-response">Device Binding Response</h4>
+            <p>
+                If the user has repeated the <code>user_code</code> for the <code>device_code</code> on time,
+                the verification URI endpoint MUST respond with a success response with a <code>200 (OK)</code> status
+                code in case the <a href="#md-device-binding">Device Binding</a> was successful; otherwise,
+                it MUST respond with an error response, as defined in Section 5.2 of [[RFC6749]]. Instead of the error
+                codes defined in Section 5.2 of [[RFC6749]], the following error codes are
+                specified for use with the <a href="#md-device-binding">Device Binding</a>:
+            </p>
+            <ul>
+                <li>
+                    <code>authorization_pending</code>: The <a href="#rp">RP</a> request is still pending as the end
+                    user hasn't yet completed the user-interaction steps. The <a href="#siop">SIOP</a> SHOULD repeat
+                    the request to the verification URI endpoint (a process known as polling). Before each new request,
+                    the <a href="#siop">SIOP</a> MUST wait at least the number of seconds specified by
+                    the <code>interval</code> parameter of the <code>&lt;SIOP DID Request&gt;</code>, or 5 seconds if
+                    none was provided, and respect any increase in the polling interval required by the
+                    <code>slow_down</code> error.
+                </li>
+                <li>
+                    <code>slow_down</code>: A variant of <code>authorization_pending</code>, the
+                    <code>&lt;Device Binding Request&gt;</code> is still pending and polling should continue, but the
+                    interval MUST be increased by 5 seconds for this and all subsequent requests.
+                </li>
+                <li>
+                    <code>expired_token</code>: The <code>device_code</code> has expired and the device binding session
+                    has concluded. The <a href="#siop">SIOP</a> SHOULD stop polling and send the
+                    <code>&lt;SIOP DID Response&gt;</code>.
```

Comment:
> send the `SIOP DID Response`

Does this imply sending a *failure* response? If so, would be good to state this explicity.